lib/mac/python/pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = 'acacia_axapi'
+version = '0.0.1'
```

Comment:
We should eventually add a top-level metadata file with version, authors, license etc. but this will do for now.